api/HTMLFieldSetElement.json

```diff
@@ -397,7 +397,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "4"
```

Comment:
`validationMessage` and the following aren't reflected content attributes, so trusting.